### DISCUSSION TITLE
Is failsafe_mission setting gone now? When was it removed?

### DISCUSSION THREAD

**Discussion by 0ldHand** [iNavFlight/inav#10237] [General]
Used to be a setting to allow waypoint missions to continue if fs is triggered during missions.

**Comment by breadoven** [comment]
Changed to `failsafe_mission_delay`. Set to -1 if you want the same behaviour as `failsafe_mission`.

**Reply by dido46** [reply]
Дякую за гілку і коментар, приємно і дуже вчасно для мене.
